patches/react-native-web-lottie+1.4.4.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/react-native-web-lottie/src/index.js b/node_modules/react-native-web-lottie/src/index.js
+index da0e363..f37104a 100644
+--- a/node_modules/react-native-web-lottie/src/index.js
++++ b/node_modules/react-native-web-lottie/src/index.js
+@@ -38,7 +38,7 @@ class Animation extends PureComponent {
+     this.anim = lottie.loadAnimation({
+       container: this.animationDOMNode,
+       animationData: props.source,
+-      renderer: 'svg',
++      renderer: 'canvas',
```

Comment:
I found this seems to yield better framerates and also fixes a display but on iOS Safari with different animation (not yet included in the app)